perfkitbenchmarker/linux_packages/hpcc.py

```diff
@@ -22,21 +22,48 @@
 
 import re
 
+from perfkitbenchmarker import data
+from perfkitbenchmarker import flags
 from perfkitbenchmarker.linux_packages import openblas
 from perfkitbenchmarker.linux_packages import INSTALL_DIR
 
+FLAGS = flags.FLAGS
+
 HPCC_TAR = 'hpcc-1.4.3.tar.gz'
 HPCC_URL = 'http://icl.cs.utk.edu/projectsfiles/hpcc/download/' + HPCC_TAR
 HPCC_DIR = '%s/hpcc-1.4.3' % INSTALL_DIR
 MAKE_FLAVOR = 'Linux_PII_CBLAS'
 HPCC_MAKEFILE = 'Make.' + MAKE_FLAVOR
 HPCC_MAKEFILE_PATH = HPCC_DIR + '/hpl/' + HPCC_MAKEFILE
 
+# A dictionary of overridable objects in this package.
+# Key is the name of of the object under perfkitbenchmarker/data directory.
+# Value is the remote location on VM.
+OVERRIDABLE = {
+    'hpcc': '.'
+}
 
-def _Install(vm):
-  """Installs the HPCC package on the VM."""
+
+def CheckPrerequisites():
+  """Verifies that required resources are present.
+
+  Raises:
+    perfkitbenchmarker.data.ResourceNotFound: On missing resource.
+  """
+  if __name__.split('.')[-1] in FLAGS.override_packages:
+    for k, _ in OVERRIDABLE.items():
+      data.ResourcePath(k)
```

Comment:
If more packages decide to be override-able, this could be reusable code. But I guess we can deal with that when the need arises.